packages/store/tests/utils/angular.spec.ts

```diff
@@ -1,82 +1,73 @@
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { getPlatform, platformCore } from '@angular/core';
-import { platformBrowser } from '@angular/platform-browser';
-import {
-  platformBrowserDynamicTesting,
-  BrowserDynamicTestingModule
-} from '@angular/platform-browser-dynamic/testing';
-import { getTestBed } from '@angular/core/testing';
+import { ɵglobal } from '@angular/core';
+
 import { isAngularInTestMode } from '@ngxs/store/internals';
 
 describe('[utils/angular]', () => {
   describe('isAngularInTestMode', () => {
-    function resetEnv() {
+    // Just an empty object so `typeof !== undefined` will be truthy.
+    const nonUndefinedValue = {};
+
+    function resetMemoization() {
       const fn = <any>isAngularInTestMode;
       // Reset the memoization
       fn && fn.reset && fn.reset();
-
-      // Reset the angular platform
-      const p = <any>getPlatform();
-      p && p.destroy && p.destroy();
     }
 
     beforeEach(() => {
-      resetEnv();
+      resetMemoization();
     });
 
     afterEach(() => {
-      resetEnv();
-
-      getTestBed().resetTestEnvironment();
-      getTestBed().initTestEnvironment(
-        BrowserDynamicTestingModule,
-        platformBrowserDynamicTesting()
-      );
+      resetMemoization();
     });
 
-    it(`should return true if the Angular Test Module has been bootstrapped`, () => {
-      // Arrange
-      platformBrowserDynamicTesting();
-      // Act
-      const result = isAngularInTestMode();
-      // Assert
-      expect(result).toEqual(true);
+    it('should return true if `__karma__` is available', () => {
+      // Arrange & act & assert
+      const __karma__ = ɵglobal.__karma__;
```

Comment:
We could have a test helper (like the `freshPlatform` one you created) that wraps the test execution with a `try..finally` and stores and restores the values on `ɵglobal` (or deletes them again if they weren't there to begin with).

Reply:
I'd like to keep these tests simple for now. We could've thought about the `isolateGlobalTestProps` implementation in the future.